engine/src/main/java/org/terasology/world/chunks/internal/ChunkImpl.java

```diff
@@ -545,12 +525,7 @@ public void prepareForReactivation() {
     public void dispose() {
         disposed = true;
         ready = false;
-        if (activeMesh != null) {
-            for (ChunkMesh chunkMesh : activeMesh) {
-                chunkMesh.dispose();
-            }
-            activeMesh = null;
-        }
+        disposeMesh();
```

Comment:
This is unrelated to the vertical segments issue, but shouldn't we dispose of the pendingMesh too, if any?